scripts/benchmarks.sh

```diff
@@ -35,7 +35,7 @@ do
 		--extrinsic='*' \
 		--steps=$steps  \
 		--repeat=$repeat \
-		--json \
+		--json-file=./bench-statemine.json \
         --header=./file_header.txt \
```

Comment:
Something with the tabs/spaces indentation here is wrong.

Reply:
good spot. fixed.